README.adoc

```diff
@@ -6,13 +6,48 @@ ____
 
 == Create a VM
 
-. launch my interactive menu
+. launch my interactive menu using the *_create_vm.sh_* script (located in the "scripts" folder)
 +
 [,bash]
 ----
 sh scripts/create_vm.sh
 ----
-. into the vm launch this
+ 
+. the interactive menu looks like this :
++
+[,bash]
+----
+main menu:
+1: setup the vm
+2: list vms
+3: exit
+4: start the vm
+5: generate info-vm.txt
+6: remove archlinux iso
+7: delete the vm
+----
+
+. select the first option to begin the setup of a new VM and follow the steps
+
+  you can skip each step if you know what you are doing
+
+** `*_create the vm_*` creates the .vbox file which contains the settings needed by VirtualBox to create the VM
+** `*_download_archlinux_iso_*` downloads and place an archlinux disk image (.iso) in the _my_vm_created_by_cli/_ folder
+** `*_attach_archlinux_iso_*` will "plug" the .iso as you would do with a bootable device to allow the VM to boot using it
+** `*_create_vdi_*` creates the _virtual disk image_ (.vdi) file where the VM is created
+** `*_create_vm_sata_*` creates the sata drive for the VM
+** `*_start_vm_*` will do what it should do
+** The following error should appear, just hit _Change Network Settings_ and hit _OK_ (default options should be fine) +
+image:images/error_net.png[,250]
+image:images/net_ok.png[,250]
+** boot up the VM +
+image:images/boot_screen.png[,250]
+image:images/VM_running.png[,250]
+
+. It can be good idea at this point to setup an SSH connection
+.. J'ai oublie comment on fait
```

Comment:
1. `ip a`
   > get the IP address of the virtual machine

2. `passwd`
   > set a default password to the `root` user. Tt is a requirement to connect by `ssh` to your live session
   
3. `ssh root@{IP_ADDRESS}`